### PR TITLE
Nep 150/add notification schema

### PR DESCRIPTION
**Pull request by shrijan-siris**

### CODE REVIEW

File: jest.setup.js [hunk 1]
```diff
@@ -1 +1 @@
-jest.setTimeout(30000)
+jest.setTimeout(9999999)
```
Comment (syuraj) [inline jest.setup.js]: @shrijan-siris I think we shouldn't increase beyond 30 seconds. A single test shouldn't take more than 10 seconds.

File: src/NotificationDbService.integration.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			article: articles[0]._id,
+			user: user._id
+		}
+
+		const savedNotification = await saveNotifications(notification)
+		expect(saveNotifications).not.toBeUndefined()
+		const deletedNotification = await deleteNotification({ _id: savedNotification[0]._id })
+		expect(deletedNotification).not.toBeUndefined()
+	})
+	// it('save multiple notifications together', async () => {
```
Comment (syuraj) [inline src/NotificationDbService.integration.js]: @shrijan-siris can you remove commented code from final pr?

File: src/NotificationDbService.integration.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+const { getUsers } = require('./UserDbService')
+const { getLatestNewsArticle } = require('./newsDbService')
+const { saveNotifications, deleteNotification } = require('./NotificationDbService')
+
+describe('Notification Db Service ', () => {
+	let articles
+	let users
+	beforeAll(async () => {
+		mongoose.connect(process.env.DATABASE_URL, { useNewUrlParser: true })
```
Comment (syuraj) [inline src/NotificationDbService.integration.js]: @shrijan-siris have you got a chance to try this?
https://dev.to/paulasantamaria/testing-node-js-mongoose-with-an-in-memory-database-32np

File: jest.setup.js (hunk 1 above)
Comment (shrijan-siris) [reply jest.setup.js]: we should get a way to test and mock for mongoose call, i was testing with 2 real data and took more than 30 seconds

File: src/NotificationDbService.integration.js (hunk 3 above)
Comment (shrijan-siris) [reply src/NotificationDbService.integration.js]: seems great

**Pull request merged**